app/resources/views/pages/location_management/locations_added_by_me.blade.php

```diff
@@ -10,27 +10,30 @@
 	@else
 		<p class="add-create"><a href="/location/management/add" title="Add New Location"><em class="fa fa-plus-circle"></em></a></p>
 	<div class="row column-titles">
-		<div class="col-xs-4">
+		<div class="col-xs-3">
 			<h3>Name</h3>
 		</div>
-		<div class="col-md-6 col-xs-4">
+		<div class="col-md-5 col-xs-3">
 			<h3>Address</h3>
 		</div>
-		<div class="col-md-2 col-xs-4">
-			<h3>Action</h3>
+		<div class="col-md-3 col-xs-3">
+			<h3>Suggestions</h3>
 		</div>
 		
 	</div>
 	<div class="data">
 		@foreach ($locations as $location)
 			<div class="row">
-				<div class="col-xs-4">
-					<a href="/location/rating/{{ $location->id }}">{{ $location->name }}</a>
+				<div class="col-xs-3">
+					<a href="/location-rating/{{ $location->id }}">{{ $location->name }}</a>
```

Comment:
It looks like some rebasing mistakes were made here with route changes that got merged Tuesday.  The new route for rating is "/location/rating/" instead of "/location-rating/" so your changes would break links in the My Locations page.

Test those other links on the "My Locations" list when you're done fixing.